apps/meteor/app/lib/server/functions/saveUser.js

```diff
@@ -1,3 +1,4 @@
+import { PasswordPolicyError } from '@rocket.chat/account-utils';
```

Comment:
I think it is worth moving this to the `utils` package instead of creating a new one, any opinion on this?